db/blip_handler.go

```diff
@@ -141,21 +141,14 @@ func collectionBlipHandler(next blipHandlerFunc) blipHandlerFunc {
 	return func(bh *blipHandler, bm *blip.Message) error {
 		collectionIndexStr, ok := bm.Properties[BlipCollection]
 		if !ok {
+			if !bh.db.hasDefaultCollection() {
+				return base.HTTPErrorf(http.StatusBadRequest, "Method requires passing a collection property and a prior GetCollections message")
+			}
```

Comment:
Should it be using the default collection and not bh.db.singleCollection below?  Or is that part of a future PR, and today "hasDefaultCollection" always means singleCollection==defaultCollection?

Reply:
It will be part of the ISGR PR forthcoming after this one.

Reply:
This is going to be fixed with ISGR PR later this week.